source/Octopus.Tentacle/Commands/RunAgentCommand.cs

```diff
@@ -70,6 +71,7 @@ public class RunAgentCommand : AbstractStandardCommand
                 // There's actually nothing to do here. The CommandHost should have already been determined before Start() was called
                 // This option is added to show help
             });
+            Options.Add("async-halibut", "Enables async Halibut support for running services asynchronously", _ => enableAsyncHalibut = true);
```

Comment:
I thought this might be a good thing to add, but if people feel like an environment variable is sufficient, then I can remove it.